opencga-analysis/src/main/java/org/opencb/opencga/analysis/alignment/qc/AlignmentQcAnalysis.java

```diff
@@ -153,13 +157,17 @@ protected List<String> getSteps() {
         if (runFastqcMetricsStep) {
             steps.add(FASTQC_METRICS_STEP);
         }
+        if (updateQcStep) {
+            steps.add(UPDATE_FILE_ALIGNMENT_QC_STEP);
+        }
         return steps;
     }
 
     @Override
     protected void run() throws ToolException {
         // Create the tool runner
-        toolRunner = new ToolRunner(opencgaHome, catalogManager, StorageEngineFactory.get(variantStorageManager.getStorageConfiguration()));
+        toolRunner = new ToolRunner(getOpencgaHome().toString(), catalogManager,
```

Comment:
You can better use the constructor 
- `public ToolRunner(String opencgaHome, CatalogManager catalogManager, VariantStorageManager variantStorageManager)` 

instead of 
- `ToolRunner(String opencgaHome, CatalogManager catalogManager, StorageEngineFactory storageEngineFactory)`